blog/templates/delete_post.html

```diff
@@ -1,19 +1,12 @@
 {% extends 'base.html'   %}
 
-{% block title %}
-    Delete Blog Post
-{% endblock %}
-
 {% block content %}
 {% if user.is_superuser %}
-    <h1>Delete Post</h1>
-    <br/><br/>
-    <h3>Delete: {{ post.title }}</h3>
-    <br/>
+    <h2>Delete: {{ post.title }}</h2>
     <div class="form-group">
         <form method="POST">
             {% csrf_token %}
-            <strong>Are you sure?</strong><br/><br/>
+            <strong>Are you sure?</strong>
```

Comment:
Good way is use 'p' tag as wrapper.
FYI:
The HTML Strong Importance Element ('strong') indicates that its contents have strong importance, seriousness, or urgency. Browsers typically render the contents in bold type.

Reply:
done